### PR TITLE
useMiddleware: add ability to specify array of middleware

### PR DESCRIPTION
**Pull request by charlierudolph**
@alexdavid @alexpeachey @kevgo

### CODE REVIEW

File: test/feature/use_middleware_helper.feature [hunk 1]
```diff
@@ -38,6 +38,38 @@ Feature: BaseController useMiddleware helper
       | GET     | /users/1 | foo           |
 
 
+  Scenario Outline: specify multiple middleware
+    Given a file "controllers/users_controller.coffee" with the content
+      """
+      {BaseController} = require 'exprestive'
+
+      middle1 = (req, res, next) ->
+        req.custom1 = 'foo'
```
Comment (kevgo) [inline test/feature/use_middleware_helper.feature:47]: What about making the content of this string "middle1 data"?

File: test/feature/use_middleware_helper.feature [hunk 2]
```diff
@@ -38,6 +38,38 @@ Feature: BaseController useMiddleware helper
       | GET     | /users/1 | foo           |
 
 
+  Scenario Outline: specify multiple middleware
```
Comment (kevgo) [inline test/feature/use_middleware_helper.feature]: specify multiple middleware**s**

**Comment by kevgo** [comment]
:ship: nice feature!

File: test/feature/use_middleware_helper.feature (hunk 2 above)
Comment (alexdavid) [inline test/feature/use_middleware_helper.feature]: middlewares is weird. How about `specify multiple middleware functions` ?

**Comment by alexdavid** [comment]
:ship:

File: test/feature/use_middleware_helper.feature (hunk 1 above)
Comment (charlierudolph) [inline test/feature/use_middleware_helper.feature:47]: Currently following the example of this [feature](https://github.com/alexdavid/exprestive/blob/b246bcff2bb913ad37fd8b0c9aaf0fba82840d2e/test/feature/middleware_support.feature#L3). Can deal with making this better in another PR.

**Pull request closed without merging**